OAI_CONFIG_LIST_sample.json

```diff
@@ -0,0 +1,21 @@
+// Please modify and rename this file into OAI_CONFIG_LIST to run the example code
```

Comment:
This file will be very helpful for users. 

I would reword the comment a bit:
// Please modify and rename this file to OAI_CONFIG_LIST to run the code examples.
or
// Please modify and rename this file to OAI_CONFIG_LIST to run the sample code.